tests/engine/storage/test_local_model_storage.py

```diff
@@ -136,6 +165,66 @@ def test_create_model_package(
     assert list(persisted_resources) == [Path(load_model_storage_dir, "resource1")]
 
 
+def test_read_unsupported_model(
+    monkeypatch: MonkeyPatch, tmp_path_factory: TempPathFactory, domain: Domain,
+):
+    train_model_storage = LocalModelStorage(
+        tmp_path_factory.mktemp("train model storage")
+    )
+    graph_schema = GraphSchema(nodes={})
+
+    persisted_model_dir = tmp_path_factory.mktemp("persisted models")
+    archive_path = persisted_model_dir / "my-model.tar.gz"
+
+    # Create outdated model meta data
+    trained_at = datetime.utcnow()
+    model_configuration = GraphModelConfiguration(
+        graph_schema, graph_schema, TrainingType.BOTH, None, None, "nlu"
+    )
+    outdated_model_meta_data = ModelMetadata(
+        trained_at=trained_at,
+        rasa_open_source_version=rasa.__version__,  # overwrite later to avoid error
+        model_id=uuid.uuid4().hex,
+        domain=domain,
+        train_schema=model_configuration.train_schema,
+        predict_schema=model_configuration.predict_schema,
+        training_type=model_configuration.training_type,
+        project_fingerprint=rasa.model.project_fingerprint(),
+        language=model_configuration.language,
+        core_target=model_configuration.core_target,
+        nlu_target=model_configuration.nlu_target,
+    )
+    old_version = "0.0.1"
+    outdated_model_meta_data.rasa_open_source_version = old_version
+
+    # Package model - and inject the outdated model meta data
+    monkeypatch.setattr(
+        LocalModelStorage,
+        "_create_model_metadata",
+        lambda *args, **kwargs: outdated_model_meta_data,
+    )
+    with freezegun.freeze_time(trained_at):
```

Comment:
why do we need the freezegun?

Reply:
nice observation 👍  has become redundant with the monkey patch